flamedisx/source.py

```diff
@@ -29,7 +31,10 @@ class Source:
     #: Whether to trace (compile into a tensorflow graph) the differential
     #: rate computation
     trace_difrate = True
+
     default_max_sigma = 3
+    default_max_dim_size = 70
```

Comment:
Could you put the default back to 120, now that the default bounds are unchanged? Or did you test that 120 is overkill even with the old bounds?

Reply:
70 was fine for the default sources, even with the old bounds

Reply:
That's good to hear, but it might affect the tests Pueh and Cristian are currently doing, and you wrote to them in the slack that the default sources would be unaffected. Since you are eager to merge quickly, can you leave the current default and open a separate PR for them to look at? @plt109 and @CristianAntochi feel free to chime in if I'm misunderstanding.

Reply:
Resolved via slack